lib/index.js

```diff
@@ -238,16 +247,50 @@ Builder.prototype.build = function(options) {
 			result.allVariables = oVariableCollector.getAllVariables();
 
 			// also compile rtl-version if requested
+			let oRTL;
 			if (options.rtl) {
-				result.cssRtl = tree.toCSS(assign(options.compiler, {
-					plugins: [new RTLPlugin()]
+				const RTLPlugin = require("./plugin/rtl");
+				oRTL = new RTLPlugin();
+			}
+
+			if (oRTL) {
```

Comment:
Why not including this into the "if" above?

Reply:
I did this to explicitly make clear that I run the RTL build only if the RTL plugin is being created. The same check exists also a bit more down.